pages/1.10/release-notes/1.10.6/index.md

```diff
@@ -50,14 +55,9 @@ DC/OS 1.10.0 includes many new capabilities for Operators and expands the collec
   
 Please try out the new features and updated data services. Provide any feedback through our support channel: <a href="https://support.mesosphere.com/">support.mesosphere.com</a>.
 
-# <a name="new-features"></a>New Features and Capabilities
-
-## Apache Mesos 1.4 and Marathon 1.5 Integrated.
-- DC/OS 1.10.0 is based on Mesos 1.4.0, here using master branch (pre-release) SHA 013f7e21, with over 1200 commits since the previous Mesos version. View the [changelog](https://github.com/apache/mesos/blob/master/CHANGELOG).
-
-- DC/OS 1.10.0 is integrated with the latest release of Marathon, version 1.5. Resulting breaking changes and new features are documented below. For more information about Marathon 1.5, consult the [Marathon changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
+## <a name="new-features"></a>New Features and Capabilities
```

Comment:
I think the HTML tags here are unnecessary and were added to fake tables of contents before they were built into the documentation site code. Remove the HTML tags from all the headings that have them.